src/AElf.Sdk.CSharp/SafeDateTime.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Sdk.CSharp
+{
+    public class SafeDateTime
```

Comment:
remove safe date time, only let smart contract use Timestamp and Duration, because DateTime has DateTime.Kind and double